frontend/src/styles/theme.ts

```diff
@@ -21,6 +22,21 @@ const theme = createTheme({
         ...typography.overline,
       },
     },
+    MuiCardHeader: {
+      root: {
+        padding: "32px 32px 16px",
+      },
+    },
+    MuiCardContent: {
+      root: {
+        padding: "16px 32px",
+      },
+    },
+    MuiCardActions: {
+      root: {
+        padding: "16px 32px",
+      },
+    },
```

Comment:
Are you sure this doesnt make something on the site more ugly?

Reply:
It's a concern, but I believe I checked most of the cards on the site with the possible exception of archive. Going to double check it before I merge.